### PR TITLE
[storybook] CommitList

### PR DESCRIPTION
**Pull request by dvdherron**
## Cute animal pic
image
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

## Link to Trello card (if applicable)
https://trello.com/c/kdSWNyII/9-metecho-commits-list
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

## Description
_The commit messages say what you did; this should explain why and/or how._
- [x ] Description is in Trello card
- Adds documentation to commit-list component

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._
Text is in the "intro" section of the component. 

## Show me
_Provide screenshots/animated gifs/videos if necessary._
image

### CODE REVIEW

**Comment by dvdherron** [comment]
[Metecho commits › list](https://trello.com/c/kdSWNyII/9-metecho-commits-list)

File: src/stories/commits/CommitList.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+changed over time.
+
+Each commit in the list features an id number, an image of the commit author,
+the commit message, and a timestamp of when the commit was made.
+
+<Story id="commits-list-component--component" />
+
+## Guidelines
+
+- _[To be added]_
```
Comment (jgerigmeyer) [inline src/stories/commits/CommitList.stories.mdx:22]: In this PR, or in a new PR?

**Review by jgerigmeyer — APPROVED** [review]

File: src/stories/commits/CommitList.stories.mdx (hunk 1 above)
Comment (dvdherron) [reply src/stories/commits/CommitList.stories.mdx:22]: @jgerigmeyer I wasn't sure if there were other notable things to mention about this component at this stage. If there isn't, I can get rid of this section.

File: src/stories/commits/CommitList.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+**Shows a history of changes on a task.**
+
+## Overview
+
+The **CommitList** component is a data table that shows the GitHub commit
+history of a **task**. This makes it possible to see at a glance how a task has
+changed over time.
+
+Each commit in the list features an id number, an image of the commit author,
```
Comment (stacyk) [inline src/stories/commits/CommitList.stories.mdx:15]: I wonder if we should say their GitHub avatar, which may not be an image of the author depending on what they are using for their avatar.

**Review by stacyk — APPROVED** [review]

File: src/stories/commits/CommitList.stories.mdx (hunk 2 above)
Comment (jgerigmeyer) [reply src/stories/commits/CommitList.stories.mdx:15]: Agreed. We could note that _all_ of this data comes directly from GitHub via a "push" webhook.

Comment (dvdherron) [reply src/stories/commits/CommitList.stories.mdx]: Updated: https://github.com/oddbird/Metecho/pull/711/commits/a27b2b51ef2cd3526d18bd65576b63611385fe81

Is this more clear @jgerigmeyer  @stacyk ?

**Review by duggiemitchell — APPROVED** [review]
LGTM!

**Pull request merged**